stripe/src/main/java/com/stripe/android/paymentsheet/ui/PrimaryButton.kt

```diff
@@ -30,16 +30,27 @@ internal class PrimaryButton @JvmOverloads constructor(
     init {
         setBackgroundResource(R.drawable.stripe_paymentsheet_buy_button_default_background)
 
+        viewBinding.label.text = getText(attrs)
+
         isClickable = true
         isEnabled = false
     }
 
-    fun setLabelText(text: String) {
-        viewBinding.label.text = text
+    private fun getText(attrs: AttributeSet?): CharSequence? {
```

Comment:
Maybe `getLabelText()`?

Reply:
It is actually get the text attribute from the attributes. I changed it to getTextAttributeValue.